R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -243,7 +243,15 @@ test_that("read csv as DataFrame", {
   expect_equal(count(withoutna2), 3)
   expect_equal(count(where(withoutna2, withoutna2$make == "Dummy")), 0)
 
+  # writing csv file
+  csvPath2 <- tempfile(pattern = "csvtest2", fileext = ".csv")
+  write.df(df2, path = csvPath2, "csv", header = "true")
+  df3 <- read.df(csvPath2, "csv", header = "true")
```

Comment:
The path is interpreted as a directory by `write.df`. It then puts in a `part-0000` or a sequence of such files inside the directory